### PR TITLE
Added jsnext:main and module field to package.json. (#15)

### PR DESCRIPTION
**Pull request by Rembane**
I hope this isn't a too weird pull request, it was too long time ago I did one last time.

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,3 +1,8 @@
 {
   "presets": ["es2015"]
+  "env": {
```
Comment (briancavalier) [inline .babelrc]: It looks like there's a missing comma at the end of line 2.  I think that's what's breaking the travis build.  Add the comma and let's see what travis says.

**Comment by briancavalier** [comment]
No too weird at all :)  Thanks for doing this, @Rembane.

**Comment by briancavalier** [comment]
Hmmm, seems travis is failing on node 4 due to peerDeps--not anything related to this PR.  Sorry about that, @Rembane.  Since node 6 is LTS now, we decided to drop node 4 from travis, but we hadn't updated this repo's travis config yet.  How about this: I'll update the travis config and get that merged to master, then you can rebase this PR on latest master so that travis will pass.  Sound ok?

**Comment by Rembane** [comment]
That sounds good. I fumled on my keys and force pushed a commit with the comma in it, so I lost the commit this pull request is based on, which probably means I have to make a new one, unless you know some magic git-fu. :)

**Comment by briancavalier** [comment]
It should be ok.  As long as the commit hash in your local branch matches the commit hash in this PR's remote branch on github (currently a160a0d), you can just push more commits to the branch and they'll show up in the PR.  If you have a different commit hash locally, but don't have any uncommitted or unpushed changes, you could just delete your local branch and re-pull it from github.

I'll update the travis config today and ping you here when that's been merged.

**Comment by briancavalier** [comment]
Ok, travis config updated (See #17).  If you rebase this on latest master, hopefully travis will approve :)

**Comment by Rembane** [comment]
I've now rebase master on jsnext, that was the only combination of rebases that preserved a160a0d. Does it look alright?

**Comment by briancavalier** [comment]
No worries @Rembane.  It should have been the other way around (rebase your jsnext onto latest master).  I merged your change in bf0b0c8, and I'll release an updated version soon.

Thanks again!

**Comment by Rembane** [comment]
@briancavalier Sweet! Now I know 'til next time! 

No worries. :)

**Pull request closed without merging**